locale/en/knowledge/advanced/buffers/how-to-use-buffers.md

```diff
@@ -13,7 +13,7 @@ layout: knowledge-post.hbs
 
 ## Why Buffers?
 
-Pure JavaScript, while great with unicode-encoded strings, does not handle straight binary data very well. This is fine on the browser, where most data is in the form of strings. However, Node.js servers have to also deal with TCP streams and reading and writing to the filesystem, both which make it necessary to deal with purely binary streams of data.
+Pure JavaScript, while great with unicode-encoded strings, does not handle straight binary data very well. This is fine on the browser, where most data is in the form of strings. However, Node.js servers have to also deal with TCP streams and reading and writing to the filesystem, both make it necessary to deal with purely binary streams of data.
```

Comment:
Not sure about this, we'll need a native English speaker to chime in

Reply:
```suggestion
Pure JavaScript, while great with unicode-encoded strings, does not handle straight binary data very well. This is fine on the browser, where most data is in the form of strings. However, Node.js servers have to also deal with TCP streams and reading and writing to the filesystem, both of which make it necessary to deal with purely binary streams of data.
```